doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc

```diff
@@ -18,7 +18,7 @@ endif::DM[]
 . Optionally, enter the number of replicas for {CENTRAL} or {CENTRAL} monitoring in the *Replicas* field. Do not change this number in a `{PRODUCT_INIT}-authoring` environment.
 . Optionally, enter requested and maximum CPU and memory limits in the fields under *Resource quotas*.
 . If you selected RH-SSO authentication, configure RH-SSO for {CENTRAL}:
-.. If you created the client for {CENTRAL}, enter the client name in the *Client name* field and the client secret in the *Client secret* field.
+.. If you created the client for {CENTRAL}, enter the client name in the *Client name* field and the client secret in the *Client secret* field. If you leave these fields blank, the deployment attempts to create a new client.
```

Comment:
_If the SSO Client does not exist the application attempts to create the client with the SSO Client, SSO Secret, HTTP and HTTPS hostnames provided_